pkgdb2/templates/package.html

```diff
@@ -75,16 +75,16 @@
     </tr>
     {% endif %}
     {% endfor %}
-    {% if is_admin %}
     <tr>
       <td colspan="2" align="center">
-        <a href="{{ url_for('.admin_log', package=package.name) }}">
+        <a href="{{ url_for('.package_logs', package=package.name) }}">
```

Comment:
Since we're renaming Logs to Timeline, maybe make this package_timeline, to stay consistent?